packages/x-components/src/x-modules/tagging/events.types.ts

```diff
@@ -26,22 +26,22 @@ export interface TaggingXEvents {
   /**
    * ClickedResultStorageKey has been configured to use the
    * {@link @empathy/x-types#Result | result} url.
-   * * Payload: The new clickedResultStorageKey.
+   * Payload: The new clickedResultStorageKey.
    */
   ResultURLTrackingEnabled: string;
   /**
    * Query tagging info has changed.
-   * * Payload: The new query tagging info.
+   * Payload: The new query tagging info.
    */
-  SearchTaggingReceived: TaggingInfo;
+  SearchTaggingReceived: TaggingRequest;
   /**
    * The user has clicked on the add to cart button from PDP.
-   * * Payload: The id of the {@link @empathy/x-types#Result | result} that the user clicked.
-   **/
+   * Payload: The id of the {@link @empathy/x-types#Result | result} that the user clicked.
```

Comment:
Shouldn't this be `@empathyco/x-types#Result`?

```suggestion
   * Payload: The id of the {@link @empathyco/x-types#Result | result} that the user clicked.
```

Reply:
Yes! good catch